src/llscan.cc

```diff
@@ -397,11 +417,50 @@ bool FindReferencesCmd::DoExecute(SBDebugger d, char** cmd,
     }
   }
 
+  delete scanner;
+
   result.SetStatus(eReturnStatusSuccessFinishResult);
   return true;
 }
 
 
+char** FindReferencesCmd::ParseScanOptions(char** cmd, ScanType& type) {
+  static struct option opts[] = {{"value", no_argument, nullptr, 'v'},
```

Comment:
Let's put a newline after first `{` and align everything nicely to the indent of 6?